packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -12,6 +12,7 @@ const SHARE_VERB = 'share';
 
 
 const propTypes = {
+  actor: PropTypes.object.isRequired,
```

Comment:
Total nitpick: would be great if this was a shape instead of an object so we know what properties the object needs.